test/e2e/kamsuite/kamsuite.go

```diff
@@ -64,6 +65,10 @@ func FeatureContext(s *godog.Suite) {
 
 	s.AfterScenario(func(*messages.Pickle, error) {
 		fmt.Println("After scenario")
+		err := ClearDir()
```

Comment:
If we are trying to delete the gitops folder `os.RemoveAll(path)`  will remove the entire folder. WDYT?

Reply:
I had tried it that way but it would remove even ```test-run``` folder and cause test failure.

Reply:
Thanks for the advice @chetan-rns , will do